hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -269,10 +290,35 @@ def _apply_updates_impl(self, delta: tu.ThreatUpdatesDelta) -> None:
                 state.pop(item.key, None)
             else:
                 state[item.key] = item
+                updated[item.key] = item
 
         self._store_state(state.values())
         self._cached_state = state
 
+        post_apply_fn(updated)
+
+    def post_apply(self, updated: t.Dict = {}):
+        """
+        After the fetcher applies an update, check to see if
+        matches for any of the updated signals and if so update
+        their tags.
+        """
+        privacy_group = self.data_s3_key.split("/")[-1].split(".")[0]
+        table = dynamodb.Table(self.data_store_table)
+
+        for update in updated.values():
+            row = update.as_csv_row()
+            # example row format: ('<signal>', '<id>', '<time added>', '<tag1 tags2>')
+            # e.g ('096a6f9...064f', 1234567891234567, '2020-07-31T18:47:45+0000', 'true_positive hma_test')
+            PDQSignalMetadata(
+                signal_id=int(row[1]),
+                ds_id=privacy_group,
+                updated_at=datetime.now(),
```

Comment:
Why not use time_added?

Reply:
because that means something completely different and is likely very source dependent. If you think it is worth propagating I can add it as its own field.

Reply:
I thought that's the same. My bad. Thanks for clarifying.